packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -85,20 +86,21 @@ class UnthemedCodeEditor extends PureComponent<Props> {
 
   handleBeforeMount = (monaco: Monaco) => {
     this.monaco = monaco;
-    const { language, getSuggestions, onBeforeEditorMount } = this.props;
 
-    if (getSuggestions) {
-      this.completionCancel = registerSuggestions(monaco, language, getSuggestions);
-    }
```

Comment:
What is the logical difference ebtween doing this onMount instead of beforeMount?

Reply:
i guess it will happen slightly later, but by doing it `onMount` we get access to the editor instance, whereas before mount we only have access to the global monaco instance